src/views/BridgeTransaction.vue

```diff
@@ -231,10 +245,18 @@ export default class BridgeTransaction extends Mixins(
     return this.historyItem?.amount ?? '';
   }
 
+  get amount2(): string {
+    return this.historyItem?.amount2 ?? this.amount;
```

Comment:
I would go for wordings like 'secondAmount', 'firstAmount', etc, instead of numerical. Not mandatory change, of course